src/scss/components/optional/_modal.scss

```diff
@@ -95,4 +95,27 @@ $modal-transitionTime    : 250ms !default;
             }
         }
 
+        .c-modal--fullpage--belowMid {
+            @include media('<mid') {
+                .c-modal-content {
+                    border-radius: 0;
+                    bottom: -100vh;
```

Comment:
`-100vh`? what effect does this have on the content?

Reply:
@simonsandersje clarified :)

Reply:
:)